omod/src/test/java/org/openmrs/module/idgen/rest/resource/IdentifierSourceResourceTest.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.idgen.rest.resource;
+
+
+import org.openmrs.module.idgen.IdentifierSource;
+import org.openmrs.module.idgen.rest.resource.IdentifierSourceResource;
+import org.openmrs.module.idgen.SequentialIdentifierGenerator;
+import org.junit.Before;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+
+
+public class IdentifierSourceResourceTest extends BaseDelegatingResourceTest<IdentifierSourceResource, IdentifierSource> {
+	
+	@Before
+	public void setup() throws Exception {
+	}
+
+	@Override
+	public String getDisplayProperty() {
+		return "identifiersource";
+	}
+
+	@Override
+	public String getUuidProperty() {
+		return null;
+	}
+
+	@Override
+	public IdentifierSource newObject() {
+		return new SequentialIdentifierGenerator();
```

Comment:
For the above, use Context.getService(IdentifierSourceService.class).getProviderByUuid(getUuidProperty());